ludwig/config_validation/checks.py

```diff
@@ -493,6 +493,9 @@ def check_llm_finetuning_trainer_config(config: "ModelConfig"):  # noqa: F821
     if config.model_type != MODEL_LLM:
         return
 
+    if config.trainer.type == "none" and config.adapter.pretrained_adapter_weights is not None:
```

Comment:
Should this be more simply

```python
if config.trainer.type == "none":
    # The NoneTrainer for ZS is valid.
    return
```

Reply:
But in this case, we would load in untrained LoRA weights if pretrained adapter weights weren't specified in the config, right? Would that be a problem?